mailpoet/mailpoet
# Feat/feels forced to allow 3rd party libs to be able to contact support - MAILPOET-4657

## Description

Update the welcome wizard and remove the dependency between loading `3rd-party libraries` and `tracking (data sharing)`



## QA notes

To access the welcome wizard, visit {Site_url}/wp-admin/admin.php?page=mailpoet-welcome-wizard e.g. http://gs.qawp.net/wp-admin/admin.php?page=mailpoet-welcome-wizard

* In the wizard, disable `3rd-party libraries`, but try to enable `data sharing`. Notice the toggle is not dependent on each other.

## Linked tickets

[MAILPOET-4657](https://mailpoet.atlassian.net/browse/MAILPOET-4657)
